backend/src/main/scala/cromwell/backend/io/WorkflowPathsBackendInitializationData.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.backend.io
+
+import java.nio.file.FileSystem
+
+import cromwell.backend.BackendInitializationData
+
+/**
+  * Extension of backend initialization data that also provides a `WorkflowPaths`, and by proxy its `List[FileSystem]`.
+  *
+  * This class consolidates code used by the legacy Local backend and the SFS backend, that need to localize data
```

Comment:
Can you spell out shared file system (assuming that's what it is, if not, it helps make my point) somewhere in here instead of the acronym?